### PR TITLE
Upgrade 11ac radios to VHT20

### LINKED ISSUE #424: VHT-Mode Support
Gluon currently only offers support for the 802.11n HT-Modes (20/40 MHz). While newer devices have gotten basic support, there is no possibility to separately specify VHT-Modes yet, which would enable 80/160 MHz channel widths for newer devices, while not breaking HT-Modes on 802.11n devices.

The newer VHT-Modes are available in OpenWRT through:
- VHT80
- VHT160

### PR DESCRIPTION
**Pull request by mweinelt**
Patch originally authored by @oleeander, tested and fixed by myself.

Fixes #424

### CODE REVIEW

File: package/gluon-core/luasrc/lib/gluon/upgrade/200-wireless [hunk 1]
```diff
@@ -20,6 +21,15 @@ local function get_channel(radio, config)
 	return channel or config.channel()
 end
 
+local function get_htmode(radio)
+       local phy = util.find_phy(uci:get_all('wireless', radio))
```
Comment (NeoRaider) [inline package/gluon-core/luasrc/lib/gluon/upgrade/200-wireless]: Hmm, I will look into getting rid of this `get_all()` by changing `iterate_radios()`.

Comment (NeoRaider) [reply package/gluon-core/luasrc/lib/gluon/upgrade/200-wireless]: `radio` is the full UCI section now, please rebase.

File: package/gluon-core/luasrc/lib/gluon/upgrade/200-wireless [hunk 2]
```diff
@@ -3,6 +3,7 @@
 local util = require 'gluon.util'
 local site = require 'gluon.site'
 local sysconfig = require 'gluon.sysconfig'
+local iwinfo = require 'iwinfo'
```
Comment (NeoRaider) [inline package/gluon-core/luasrc/lib/gluon/upgrade/200-wireless]: Missing dependency of *gluon-core* on *libiwinfo-lua*.

**Comment by mweinelt** [comment]
Updated as I understand the intent of your change. Will still need to test those.

**Comment by NeoRaider** [comment]
LGTM, will wait with the merge until it has been tested.

**Comment by mweinelt** [comment]
Tested and working.

**Pull request merged**